SPECS/iotop/iotop.spec

```diff
@@ -1,64 +1,75 @@
-%{!?python2_sitelib: %global python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-
-Summary:	Iotop is a Python program with a top like UI used to show the processes and their corresponding IO activity.
-Name:		iotop
-Version:	0.6
-Release:        8%{?dist}
-License:	GPLv2
-URL:		http://guichaz.free.fr/iotop/
-Group:		System/Monitoring
+Summary:        Iotop is a Python program with a top like UI used to show the processes and their corresponding IO activity.
+Name:           iotop
+Version:        0.6
+Release:        9%{?dist}
+License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:	http://guichaz.free.fr/iotop/files/%{name}-%{version}.tar.gz
-BuildRequires: python2 python2-libs
-Requires:       python2
-Requires:       python2-libs
-
+Group:          System/Monitoring
+URL:            http://guichaz.free.fr/iotop/
```

Comment:
Nit: https

Reply:
Same for Source

Reply:
Site doesn't load for me when using HTTPS, sadly.